.github/workflows/release-workflow.yml

```diff
@@ -0,0 +1,86 @@
+name: Release workflow
+# This workflow is triggered on creating tags to master or a opendistro release branch
+on:
+  push:
```

Comment:
Did you come up with which was better between push and create? Can you share any info you read on why choosing one over the other?

Reply:
I read that some people faced issue with push but then the security plugin is tried and tested with this one so I guess it should work fine